.github/workflows/golangci-lint.yml

```
@@ -30,6 +30,7 @@ jobs:
           go-version-file: 'go.mod'
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v3
+        if: github.event_name == 'push' || (github.event_name == 'pull_request' && startsWith(github.head_ref, 'release/'))
```

Does this need a `!`?
```suggestion
        if: github.event_name == 'push' || (github.event_name == 'pull_request' && !startsWith(github.head_ref, 'release/'))
```

---

Good catch, thanks!